playbooks/installation/installation.adoc

```diff
@@ -22,21 +23,53 @@ If adding those entries is problematic, a quick DNS solution for OpenShift is to
 
 === Node Sizing
 
-The Offcial Documentation lists some link:https://docs.openshift.com/enterprise/3.0/admin_guide/install/prerequisites.html#system-requirements[minimal system requirements] for OpenShift masters and nodes. Beyond that, there are several things to consider when deciding on the sizing for your nodes.
+|===============
+| Masters |
+Physical or virtual system, or an instance running on a public or private IaaS.
+
+Base OS: RHEL 7.1 or later with "Minimal" installation option, or RHEL Atomic Host 7.2.4 or later.
+
+2 vCPU.
+
+Minimum 8 GB RAM.
+
+Minimum 30 GB hard disk space for the file system containing /var/.
+
+| Nodes |
+Physical or virtual system, or an instance running on a public or private IaaS.
+
+Base OS: RHEL 7.1 or later with "Minimal" installation option, or RHEL Atomic Host 7.2.4 or later.
+
+NetworkManager 1.0 or later
+
+1 vCPU.
+
+Minimum 8 GB RAM.
+
+Minimum 15 GB hard disk space for the file system containing /var/.
+
+An additional minimum 15 GB unallocated space to be used for Docker’s storage back end; see Configuring Docker Storage below.
+|===============
```

Comment:
This is essentially a the same as the minimum system requirements found in the installation prerequisites. Should just link to prerequisites instead.